### DISCUSSION TITLE
How do I disable IPv6?

### DISCUSSION THREAD

**Discussion by AlexMini2517** [schwabe/ics-openvpn#1518] [Configuration problems]
To make issues more manageable, I would appreciate it if you fill out the following details as applicable:

# General information
1. Android Version: 12
2. Android Vendor/Custom ROM: MIUI
3. Version of the app (version number/play store version/self-built): 0.7.38
 
# Description of the issue
I have an IPv6 leak and I would like to fix it.

![image](https://user-images.githubusercontent.com/76444133/183495751-eae5a009-86a2-480a-a4c1-4af1c7bbcf49.png)

**Comment by schwabe** [comment]
CHeck the option to disable IP protocols that the VPN does not use or switch to a VPN provider that offers IPv6 connectivity

**Reply by AlexMini2517** [reply]
Where do I find this option?

**Reply by schwabe** [reply]
Under routing

**Reply by AlexMini2517** [reply]
Could you please tell me precisely what option to enable/disable?

**Reply by AlexMini2517** [reply]
![IMG_20220808_235914](https://user-images.githubusercontent.com/76444133/183521949-c01af352-7efc-437c-a162-4bfbb29e972e.jpg)

I have this enabled but I can see my IPv6 anyway. What should I do?

**Reply by schwabe** [reply]
What is your Android phone? Might be one of the broken ones.

**Reply by AlexMini2517** [reply]
Xiaomi Mi 10T Pro

**Reply by schwabe** [reply]
If that option is checked, your VPN provider not pushing IPv6 and you have a IPv6 leak then this is broken in your phone. There is nothing the app can do about that.

**Reply by AlexMini2517** [reply]
What can I do? Is there something in the system settings I can try?
